boards/nxp/mimxrt700_evk/doc/index.rst

```diff
@@ -0,0 +1,200 @@
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| IOCON     | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| CLOCK     | on-chip    | clock_control                       |
++-----------+------------+-------------------------------------+
+
+The default configuration can be found in the defconfig file:
+
+   :zephyr_file:`boards/nxp/mimxrt700_evk/mimxrt700_evk_MIMXRT798S_CM33_CPU0_defconfig`
+   :zephyr_file:`boards/nxp/mimxrt700_evk/mimxrt700_evk_MIMXRT798S_CM33_CPU1_defconfig`
+
+Other hardware features are not currently supported by the port.
+
+Connections and IOs
+===================
+
+The MIMXRT798 SoC has IOCON registers, which can be used to configure the
+functionality of a pin.
+
++---------+-----------------+----------------------------+
+| Name    | Function        | Usage                      |
++=========+=================+============================+
+| PIO0_31 | UART0           | UART RX                    |
++---------+-----------------+----------------------------+
+| PIO1_0  | UART0           | UART TX                    |
++---------+-----------------+----------------------------+
+| PIO0_18 | GPIO            | GREEN LED                  |
++---------+-----------------+----------------------------+
+| PIO0_9  | GPIO            | SW5                        |
++---------+-----------------+----------------------------+
+| PIO8_14 | UART19          | UART TX                    |
++---------+-----------------+----------------------------+
+| PIO8_15 | UART19          | UART RX                    |
++---------+-----------------+----------------------------+
+
+System Clock
+============
+
+The MIMXRT700 EVK is configured to use the Systick
+as a source for the system clock.
+
+Programming and Debugging
+*************************
+
+Build and flash applications as usual (see :ref:`build_an_application` and
+:ref:`application_run` for more details).
+
+Configuring a Debug Probe
+=========================
+
+A debug probe is used for both flashing and debugging the board. This board is
+configured by default to use the MCU-Link CMSIS-DAP Onboard Debug Probe.
+
+Using LinkServer
+----------------
+
+Linkserver is the default runner for this board, and supports the factory
```

Comment:
```suggestion
The Linkserver runner supports the factory
```
JLink is the default runner in the PR.  We need to correct this.

Reply:
Updated, thank you for reminding.